policy/release/test.rego

```diff
@@ -227,6 +228,34 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# METADATA
+# title: Image digest is present in IMAGES_PROCESSED result
+# description: >-
+#   Ensure that task producing the IMAGES_PROCESSED result contains the
+#   digests of the built image.
+# custom:
+#   short_name: test_all_images
+#   failure_msg: Test '%s' did not process image with digest '%s'.
+#   solution: >-
+#     Found an image not processed by a task. Make sure that the task
+#     processes and includes the image digest of all images in the
+#     `IMAGES_PROCESSED` result.
+#   collections:
+#   - redhat
+#
```

Comment:
Let's put an effective_on, so the rule is a warning at first and after this date a violation
```suggestion
#   effective_on: 2024-05-29T00:00:00Z
#
```

Reply:
Good idea! Done